tests/e2e/integ-suite-kind.sh

```diff
@@ -53,7 +53,7 @@ function setup_kind_registry() {
 
   # https://docs.tilt.dev/choosing_clusters.html#discovering-the-registry
   # TODO get context/config from existing variables
-  kind export kubeconfig --name="${KIND_CLUSTER_NAME}"
+  kind export kubeconfig --name="${KIND_CLUSTER_NAME}" --kubeconfig="${HOME}/.kube/config"
```

Comment:
@luksa this is to be able to locate later the kubeconfig in a know place

Reply:
Is this necessary? Isn't `~/.kube/config` the default location?

Reply:
Yes, I did the test without setting the path where we want to place the Kube config and that was not the path where the kind cluster was placing the file

Reply:
Perhaps the `KUBECONFIG` env var is set? If you check `kind export kubeconfig --help`, it says that it writes to `$HOME/.kube/config` or `$KUBECONFIG` by default.

Reply:
Yes, `  KUBECONFIG_DIR="${ARTIFACTS:-$(mktemp -d)}/kubeconfig"` we have this. Um let me see How can I change this